README.md

```diff
@@ -69,6 +69,8 @@ $ klocust apply hello -cf hello-klocust.yaml -lf hello-locustfile.yaml
 $ klocust delete hello
 ```
 
+[build from source]: ./docs/build_from_source.md
 
 
-[build from source]: ./docs/build_from_source.md
+## Contribution Guide
```

Comment:
Could you move `Contribution Guide` upper to `build from source` page link? (Sorry, I don't know right name of that).

Reply:
Oh Okay! You already made this. I will delete CONTRIBUTING.md and link build_from_source.md to it

Reply:
No... I did something stupid... Never mind.

I will change the listing order